tests/buyers/helpers/test_ods.py

```diff
@@ -200,15 +196,14 @@ def test_add_font(self):
 
         instance.add_font(fontface)
 
-        instance._document.fontfacedecls.addElement\
-                .assert_called_once_with(fontface)
+        instance._document.fontfacedecls.addElement.assert_called_once_with(fontface)
 
-    def save(self):
+    def test_save(self):
```

Comment:
Flake8 caught the `NameError: name 'fontface' is not defined`
I realised that it was a copy paste error and straightened up the test